dw-select.js

```diff
@@ -376,6 +388,36 @@ export class DwSelect extends LitElement {
        */
       _overlay: { type: Boolean, reflect: true,  attribute: 'overlay'},
 
+      /**
+       * default iconsize is 24
+       */
+      backIconSize: { type: String }, 

```

Comment:
Main thing, I want to confirm here is "Is this backward compatible change?", If no, we should release a new major version